docs/UsersGuide/source/InputOutputFiles.rst

```diff
@@ -325,8 +325,10 @@ Default Initial and Lateral Boundary Conditions
 -----------------------------------------------
 The default initial and lateral boundary condition files are set to be a severe weather case
 from 20190615 at 00 UTC. FV3GFS grib2 files are the default model and file format. A tar file
-containing the model data for this case is available on the FTP data repository at:
-https://ftp.emc.ncep.noaa.gov/EIB/UFS/SRW/v1p0/simple_test_case/. 
+(``gst_model_data.tar.gz``) containing the model data for this case is available on EMC's FTP 
+data repository at https://ftp.emc.ncep.noaa.gov/EIB/UFS/SRW/v1p0/simple_test_case/. It is 
+also available on Amazon Web Services (AWS) at 
+https://ufs-data.s3.amazonaws.com/public_release/ufs-srweather-app-v1.0.0/ic/gst_model_data.tar.gz.
```

Comment:
These files are also linked at the end of section 7.3.2 so we  are well covered.

Reply:
One more request. Please add the following to the graphics chapter (in  bold below). 

"The **full set  of** Cartopy shape files can be downloaded at https://www.naturalearthdata.com/downloads/. **For ease, the small subset of files required for these Python scripts can be obtained from the EMC ftp data repository **(linked to: https://ftp.emc.ncep.noaa.gov/EIB/UFS/SRW/v1p0/natural_earth/natural_earth_ufs-srw-release-v1.0.0.tar.gz)** or from AWS cloud storage **(linked  to https://ufs-data.s3.amazonaws.com/public_release/ufs-srweather-app-v1.0.0/natural_earth/natural_earth_ufs-srw-release-v1.0.0.tar.gz)**.** The medium scale..."